mobile/lib/services/upload.service.dart

```diff
@@ -316,7 +321,7 @@ class UploadService {
     final url = Uri.parse('$serverEndpoint/assets').toString();
     final headers = ApiService.getRequestHeaders();
     final deviceId = Store.get(StoreKey.deviceId);
-
+    final requiresWiFi = _appSettingsService.getSetting(AppSettingsEnum.uploadRequredWifi);
```

Comment:
Changing to the `SettingService` also allows us to do this


```suggestion
    final requiresWiFi = AppSetting.get(Setting.uploadRequredWifi);
```